api/v1/db/json/notes-dao.js

```diff
@@ -73,6 +73,35 @@ const getNotes = query => new Promise((resolve, reject) => {
   }
 });
 
+/**
+ * @summary Fetch a note from the database by its noteID
+ * @function
+ * @param noteID
+ * @returns {Object} The raw note from the DB
+ */
+const fetchNote = (noteID) => {
+  try {
+    const studentID = noteID.split('-')[0];
+    const studentDirPath = `${dbDirectoryPath}/${studentID}`;
+    return JSON.parse(fs.readFileSync(`${studentDirPath}/${noteID}.json`, 'utf8'));
+  } catch (err) {
+    return null;
+  }
+};
+
+/**
+ * @summary Write newContents to the note with id noteID
+ * @function
+ * @param noteID
+ * @param newContents
+ * @param options Additional options for writeFileSync
+ */
+const writeNote = (noteID, newContents, options = {}) => {
+  const studentID = noteID.split('-')[0];
```

Comment:
This parsing logic appears in all /notes/{noteID} methods. I'm thinking we should move this logic to the path file only once instead of in every method in DAO file.

Reply:
From our discussion, I moved this logic to a simple function in `notes-dao.js`